rootfs/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/deis/base:0.3.0
+
+ENV VERSION=0.3.8.linux-amd64.go1.6.2
```

Comment:
Nit... this will set `VERSION` not only for the following steps, but also in the final image-- where it won't be referenced ever again. It might just make more sense to `RUN export VERSION=0.3.8.linux-amd64.go1.6.2 \` and then combine it with the two `RUN` directives below.